src/Instance/ItemsList/ItemsList.js

```diff
@@ -107,6 +107,35 @@ const visibleColumns = [
 const dragVisibleColumns = ['dnd', 'select', ...visibleColumns];
 const rowMetadata = ['id', 'holdingsRecordId'];
 
+const DropZone = ({
+  isItemsDropable,
+  children,
+  droppableId,
+  isDropDisabled,
+}) => {
+  return isItemsDropable ? (
```

Comment:
could you explain why?

Reply:
2 Droppable in one DnD context on split-screen can conflict, and when we move holdings it's a problem

Reply:
is it disabled when holding move in progress?